### PR TITLE
Add support for GlueSchemaRegistry message deserialization.

### PR DESCRIPTION
**Pull request by blacktooth**
*Issue #, if available:* https://github.com/awslabs/amazon-kinesis-client/issues/753

*Description of changes:*

Adding support for Schema Registry functionality in KCL.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: amazon-kinesis-client/src/main/java/software/amazon/kinesis/lifecycle/ShardConsumerArgument.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 package software.amazon.kinesis.lifecycle;
 
+import com.amazonaws.services.schemaregistry.deserializers.GlueSchemaRegistryDeserializer;
```
Comment (ashwing) [inline amazon-kinesis-client/src/main/java/software/amazon/kinesis/lifecycle/ShardConsumerArgument.java:18]: Can we remove this?

**Review by ashwing — COMMENTED** [review]
Just a minor comment. Otherwise changes are inline with internal CR.

File: amazon-kinesis-client/src/main/java/software/amazon/kinesis/lifecycle/ShardConsumerArgument.java (hunk 1 above)
Comment (blacktooth) [reply amazon-kinesis-client/src/main/java/software/amazon/kinesis/lifecycle/ShardConsumerArgument.java]: Removed.

**Comment by blacktooth** [comment]
Updated the PR with dependency library path change and removed unused import.

**Review by ashwing — APPROVED** [review]
lgtm

**Comment by blacktooth** [comment]
Updating the dependencies.

**Review by ashwing — APPROVED** [review]
lgtm

**Pull request merged**